src/js/App/RootApp.js

```diff
@@ -109,10 +111,33 @@ RootApp.propTypes = {
 };
 
 const ScalprumRoot = ({ config, ...props }) => {
+  const [activeQuickStartID, setActiveQuickStartID] = React.useState('');
+  const [allQuickStartStates, setAllQuickStartStates] = useLocalStorage('insights-quickstarts', {});
+  const valuesForQuickstartContext = useValuesForQuickStartContext({
+    activeQuickStartID,
+    setActiveQuickStartID,
+    allQuickStartStates,
+    setAllQuickStartStates,
+  });
+
   return (
-    <ScalprumProvider config={config} api={{ chrome: { experimentalApi: true } }}>
-      <RootApp {...props} />
-    </ScalprumProvider>
+    <QuickStartContext.Provider value={valuesForQuickstartContext}>
+      <QuickStartDrawer>
+        <ScalprumProvider
+          config={config}
+          api={{
+            chrome: {
+              experimentalApi: true,
+              setQuickStarts: valuesForQuickstartContext.setAllQuickStarts,
```

Comment:
thoughts on API? Should we nest them? i.e. could be something like
```
chrome: {
  experimentalApi: true,
  quickStarts: {
    set: valuesForQuickstartContext.setAllQuickStarts,
    toggle: valuesForQuickstartContext.setActiveQuickStart.
    Catalog: QuickStartCatalogPage
  }
}
```

Reply:
Definitely nest the API, we can create specific hooks like `useQuickstarts` to logically separate them. I would even suggest having it under a separate key (move it out of the chrome object).